### DISCUSSION TITLE
how to print some information

### DISCUSSION THREAD

**Discussion by Alma593** [sarchlab/mgpusim#83] [General]
how can i print out the latency of each L1 cache access in the console while running benchmark? Where in the code should I make the change

**Comment by syifan** [comment]
Do you want to record the average latency on each L1 cache or the latency of each L1 cache transaction?

**Comment by Alma593** [comment]
the latency of each L1 cache transaction
Does it mean that I need to modify the code in AverageTimeTracer.go?

**Comment by syifan** [comment]
That is a good place to add a print. Or you can just write a similar tracer on your own to prevent printing too many things.

**Comment by Alma593** [comment]
Thanks for the tip!
